### DISCUSSION TITLE
Why does netavark's firewalld zone include "zone" in its name?

### DISCUSSION THREAD

**Discussion by unknowndevQwQ** [containers/netavark#668] [Q&A] [answered]
The zone added by docker in firewalld is named docker instead of docker_zone, and I would like to know the reason why netavark names the added zone as netavark_zone instead of netavark.

**Comment by mheon** [comment, marked as answer]
I don't think there is a reason. Is this important?

**Reply by unknowndevQwQ** [reply]
> I don't think there is a reason. Is this important?

I just want to know the reason for netavark to do this, no other ideas.
